client/jetpack-connect/remote-credentials.js

```diff
@@ -0,0 +1,221 @@
+		}
+		this.setState( { submitting: true } );
+
+		const url = siteToConnect;
+		const username = this.state.username;
+		const password = this.state.password;
+
+		this.props.jetpackRemoteInstall( url, username, password );
+
+		return;
+	};
+
+	componentDidUpdate() {
+		const { installError, isResponseCompleted, siteToConnect } = this.props;
+
+		if ( isResponseCompleted ) {
+			externalRedirect( addCalypsoEnvQueryArg( siteToConnect + REMOTE_PATH_AUTH ) );
+		}
+		if ( installError ) {
+			//handle errors
+		}
+	}
+
+	getChangeHandler = field => event => {
+		this.setState( { [ field ]: event.target.value } );
+	};
+	getHeaderImage() {
+		return (
+			<div>
+				<JetpackLogo full size={ 45 } />
+			</div>
+		);
+	}
+
+	getHeaderText() {
+		const { translate } = this.props;
+
+		return translate( 'Add your website credentials' );
+	}
+
+	getSubHeaderText() {
+		const { translate } = this.props;
+
+		return (
+			<div className="jetpack-connect__install-step">
+				{ translate(
+					'Add your WordPress administrator credentials ' +
+						'for this site. Your credentials will not be stored and are used for the purpose' +
+						'of installing Jetpack securely. You can also skip this step entirely and install Jetpack manually.'
+				) }
+			</div>
+		);
+	}
+
+	formFields() {
+		return (
+			<div>
+				<FormLabel htmlFor="username">{ this.props.translate( 'Username' ) }</FormLabel>
+				<div className="jetpack-connect__site-address-container">
+					<Gridicon size={ 24 } icon="user" />
+					<FormTextInput
+						autoCapitalize="off"
+						autoCorrect="off"
+						className="credentials--form__input"
+						disabled={ this.state.submitting }
+						id="username"
+						name="username"
+						onChange={ this.getChangeHandler( 'username' ) }
+						value={ this.state.username || '' }
+					/>
+				</div>
+
+				<FormLabel htmlFor="password">{ this.props.translate( 'Password' ) }</FormLabel>
+				<div className="jetpack-connect__site-address-container">
+					<div className="jetpack-connect__password-form">
+						<Gridicon size={ 24 } icon="lock" />
+						<FormPasswordInput
+							className="credentials--form__input"
+							disabled={ this.state.submitting }
+							id="password"
+							name="password"
+							onChange={ this.getChangeHandler( 'password' ) }
+							value={ this.state.password || '' }
+						/>
+					</div>
+				</div>
+			</div>
+		);
+	}
+
+	formFooter() {
+		const { translate } = this.props;
+		return (
+			<FormButton
+				className="credentials--form__submit"
+				disabled={ ! this.state.username || ! this.state.password }
+			>
+				{ translate( 'Install Jetpack' ) }
+			</FormButton>
+		);
+	}
+
+	formHeader() {
+		return (
+			<div>
+				{ this.getHeaderImage() }
+				{ this.renderHeadersText() }
+			</div>
+		);
+	}
+
+	handleHelpButtonClick() {
+		//do something
+	}
+
+	footerLink() {
+		return (
+			<LoggedOutFormFooter>
+				<LoggedOutFormLinkItem>
+					{ this.props.translate( 'Install Jetpack manually.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Install Jetpack manually' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).